db/migrate/20150701173332_extend_vmt_to_tenant_relationship.rb

```diff
@@ -0,0 +1,33 @@
+class ExtendVmtToTenantRelationship < ActiveRecord::Migration
+  def up
+    # Spawn linking table
+    create_table :atmosphere_virtual_machine_template_tenants do |t|
+      t.references :virtual_machine_template
+      t.references :tenant
+    end
+    # Rewrite existing data
+    existing_templates = execute("SELECT id, tenant_id FROM atmosphere_virtual_machine_templates")
+    existing_templates.each do |t|
+      puts "Processing VMT #{t['id'].to_s} which belongs to tenant #{t['tenant_id'].to_s}"
+      execute("INSERT INTO atmosphere_virtual_machine_template_tenants(virtual_machine_template_id, \
+        tenant_id) VALUES(#{t['id'].to_s}, #{t['tenant_id'].to_s})")
```

Comment:
What's an "sql save version"? ;)

Seriously, though: `execute` accepts a string argument and is not aware of any rails-specific interpolation mechanisms. I take it you're worried about SQL injection but we are dealing **numerical identifiers** which I have only just read from **our very own database**. What you propose could be done with `ActiveRecord::Sanitization.sanitize_sql` but I respectfully refuse to do it as I believe this is pure tinfoil-hat paranoia and needless obfuscation of code. ;)